packages/client/hmi-client/src/components/data-explorer/search-bar.vue

```diff
@@ -1,22 +1,37 @@
 <template>
 	<div class="search-bar-container">
-		<span class="p-input-icon-left p-input-icon-right">
-			<i class="pi pi-search" />
-			<InputText
-				type="text"
-				class="input-text"
-				placeholder="Search"
-				v-model="searchText"
-				@keyup.enter="addSearchTerm"
-			/>
+		<div class="search">
+			<span class="p-input-icon-left p-input-icon-right">
+				<i class="pi pi-search" />
+				<InputText
+					ref="inputElement"
+					type="text"
+					placeholder="Search"
+					v-model="searchText"
+					@keyup.enter="execSearch"
+				/>
+				<i
+					class="pi pi-times clear-search"
+					:class="{ hidden: isClearSearchButtonHidden }"
+					style="font-size: 1rem"
+					@click="clearText"
+				></i>
+			</span>
+			<i class="pi pi-history" />
+		</div>
+		<div class="suggested-items" v-if="isSuggestedItemsVisible">
+			<span>Suggested items</span>
+			<ul>
+				<li v-for="item in suggestedItems" :key="item">
+					<span class="item" @click="addSearchTerm(item)">{{ item }}</span>
```

Comment:
They should be `<Button :label="item" class="p-button-text" />` if they provide an user action.